pkg/build/api/v1beta3/types.go

```diff
@@ -632,6 +715,27 @@ type BuildRequest struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar `json:"env,omitempty"`
+
+	// BuildCauses can pass information from the controllers to the buildconfig
+	// generator about who/what has triggered a build through the request.
+	BuildCauses []*BuildTriggerCauses `json:"BuildCauses,omitempty"`
```

Comment:
This shouldn't be called `BuildCauses` - `Causes` means something else.  This should be `TriggeredBy`